javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -236,11 +235,12 @@ public Object slang_persistent() {
         }
     }
 
+    /** Aggregated, randomized access to every element */
     public static class Get extends Base {
         @Benchmark
         public int java_mutable() {
             int aggregate = 0;
-            for (int i = 0; i < CONTAINER_SIZE; i++) {
+            for (int i : RANDOMIZED_INDICES) {
```

Comment:
random access is what we're after, not linear (Scala optimizes for linear, by storing the previous element. I don't think we want that.)